Geovation Community Map/community_map.js

```diff
@@ -44,45 +46,18 @@ let style = {
 
 // Initialize the map object.
 let map = new mapboxgl.Map({
```

Comment:
you may consider to add this to addMapFeatures instead (which would be renamed to createMap

so you would have 
`const map = createMap()`